### PR TITLE
#130: unique room numbers

### PR DESCRIPTION
**Pull request by CarenRizk**
fixed some nonunique room codes and added a unique identifier to the room's model

### CODE REVIEW

File: app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java [hunk 1]
```diff
@@ -311,7 +311,8 @@ public boolean setupClassMarkerClickListener(GoogleMap map) {
         map.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
             @Override
             public boolean onMarkerClick(Marker marker) {
-                Log.i(TAG,marker.getTag().toString());
+                System.out.println(marker.getPosition());
```
Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java]: Don't we prefer logs over system out prints?

File: app/src/main/res/raw/rooms_info.json [hunk 2]
```diff
@@ -297,111 +297,111 @@
...
+      "roomCode":"842",
       "floorCode":"H-8",
       "centerCoordinates":{
         "longitude":-73.5789343714714,
         "latitude":45.49737026836251
       }
     },
     {
-      "roomCode":"elevator",
+      "roomCode":"elevators-2",
```
Comment (viveanban) [inline app/src/main/res/raw/rooms_info.json]: Are we not distinguishing staff elevators from elevators accessible to everyone?

File: app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java (hunk 1 above)
Comment (CarenRizk) [reply app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java]: yeah I forgot to remove it :s, I'll change it now

File: app/src/main/res/raw/rooms_info.json (hunk 2 above)
Comment (CarenRizk) [reply app/src/main/res/raw/rooms_info.json]: Fixed to have two different types, "elevators-#" and "staff elevators-#"

**Pull request merged**